tests/core/data/utilities/test_loading.py

```diff
@@ -149,6 +151,13 @@ def test_load_data_frame(tmpdir, extension, write):
             Image.Image,
             marks=pytest.mark.skipif(not _IMAGE_TESTING, reason="image libraries aren't installed."),
         ),
+        # it shouldn't try to expand glob patterns in URLs
+        pytest.param(
+            "https://39.yt/ant_1 [test].jpg",
```

Comment:
@Borda, can you please move this to `https://pl-flash-data.s3.amazonaws.com/images/`?

Reply:
what do you mean, what is the image or where I can find it?
probably the simples would be as part of CI just to copy one of the existing ants and rename the new file? :chipmunk:

Reply:
@Borda, I want to test that it can load a file named like this from an URL. URLs require a different kind of escaping.

You can download the image from here: https://39.yt/ant_1 [test].jpg. The point is to preserve the image name when uploading it elsewhere.

Reply:
@yurijmikhalevich uploaded to https://pl-flash-data.s3.amazonaws.com/images/ant_1+%5Btest%5D.jpg 

Unfortunately, even though the name has been preserved it does not reflect in the URL.

Reply:
@justusschock, can you make it public, please?